Rakefile

```diff
@@ -14,3 +14,27 @@ desc "Build and push gem to RubyGems without pushing to source control"
 task "release", [:remote] => ["build", "release:guard_clean", "release:rubygem_push"] do
   puts "Built and pushed gem to RubyGems without pushing to source control."
 end
+
+desc "Run normal tests (exclude no_missing_require_* specs)"
+task :test_normal do
+  sh "bundle exec rspec --exclude-pattern 'spec/line/bot/line_bot_api_gem_spec.rb,spec/line/bot/line_bot_gem_spec.rb'"
```

Comment:
`--exclude-pattern` allows multiple target when we use comma like `--exclude-pattern 'A,B,C'`, though `--exclude-pattern 'A' --exclude-pattern 'B'` doesn't work.